slave/process-fs-scratch-local/changelog

```diff
@@ -1,3 +1,10 @@
+perun-slave-process-fs-scratch-local (3.1.5) stable; urgency=low
+
+  * Modify service behaviour to be able to continue from it's own failure
```

Comment:
This is not comprehensible. You probably want to say it can recover from a failure in the previous run.